src/views/OverviewPage.vue

```diff
@@ -139,6 +188,21 @@ export default {
           alert(error);
         });
     },
+    fetchNotes() {
+      const vm = this;
+      var notesQuery = new AV.Query("Note");
```

Comment:
Use `const` whenever you can.

See https://leanpub.com/understandinges6/read#leanpub-auto-emerging-best-practices-for-block-bindings

Reply:
Also the docs of LeanStorage just got updated with all code examples changed to ES6 style. You can refresh the page to see the latest version.

Reply:
fixed